src/Mailer/Transport/SimpleQueueTransport.php

```diff
@@ -49,6 +49,12 @@ public function send(Email $email) {
 			'template' => $email->template(), //template() gives 2 values - template and layout
 			'viewVars' => [$email->viewVars()]
 		];
+		
+		foreach ($settings as $setting => $value) {
+			if (empty($value) || array_key_exists(0, $value) && empty($value[0])) {
```

Comment:
`pr($email->replyTo());` gives `array()` and then when it tries recreate email in https://github.com/dereuromark/cakephp-queue/blob/master/src/Shell/Task/QueueEmailTask.php#L118-L120
it gives
`call_user_func_array([$email, 'replyTo'], (array)[[]]);` so it thinks its more than 1 mail IMHO message is unclear and should say that its expecting value => key pair as in https://github.com/cakephp/cakephp/blob/master/src/Mailer/Email.php#L400

 it doesnt give error on tests because email is never recreated there